lib/model/om/BaseTerm.php

```diff
@@ -109,31 +109,6 @@ public function __isset($name)
     {
     }
 
-    if ('accessionsRelatedByacquisitionTypeId' == $name)
```

Comment:
(Save as previous comment.)

Reply:
Fixed.

Reply:
Based on our discussion below, I have reverted my fix to re-order these items.